Packs/DelineaSS/Integrations/DelineaSS/DelineaSS.py

```diff
@@ -476,7 +477,60 @@ def secret_rpc_changepassword_command(client, secret_id: str = '', newpassword:
     )
 
 
-def main():  # pragma: no cover
+def user_fetch_command(client, secretids):
+    credentials = fetch_credentials(client, secretids)
+    markdown = tableToMarkdown('Credentials to fetch', credentials)
+
+    return CommandResults(
+        readable_output=markdown,
+        outputs_prefix="Delinea.User.Fetch.Credentials",
+        outputs_key_field="credentials",
+        raw_response=credentials,
+        outputs=credentials
+    )
+
+
+def fetch_credentials(client, secretids):
+    credentials: List[Any] = []
+    finalsecretsid: List[Any] = []
+    finalcredentials: List[Any] = []
+    try:
+        secretsid = argToList(secretids)
+    except Exception as e:
+        demisto.debug(f"Could not fetch credentials: Provide valid secret id.{e}")
+        credentials = []
+
+    for id in secretsid:
+        if id not in finalsecretsid:
+            finalsecretsid.append(id)
+
+    if len(finalsecretsid) == 0:
+        credentials.clear()
+        demisto.credentials(credentials)

```

Comment:
why adding these lines? I think once demisto.credentials() gets an empty list it does that automatically, and since you set credentials = [] in this case (next lines) then you can just call it outside of the if statment.

Reply:
Removed the code.